darshan-util/pydarshan/pyproject.toml

```diff
@@ -3,3 +3,26 @@ requires = [
     "wheel",
     "setuptools",
 ]
+
+[tool.cibuildwheel]
+before-all = [
+    "yum install -y blas-devel lapack-devel",
+    "yum install -y libxslt-devel libxml2-devel",
+    "yum install -y libjpeg libjpeg-devel",
+    "git submodule update --init",
+    "./prepare.sh",
+    "./configure --disable-darshan-runtime --enable-apxc-mod --enable-apmpi-mod",
+    "make install"
+]
+environment = "PYDARSHAN_BUILD_EXT=1"
+skip = [
+    "*musllinux*",
+    "*i686*"
+]
+test-requires = [
+    "pytest",
+    "lxml",
+    "matplotlib<3.5",
+    "importlib_resources;python_version<'3.9'"
+]
```

Comment:
It may be worth a comment or issue or something like that to remind us to make the logs repo available for testing here eventually, so that wheels get a thorough testing treatment before release.

We can't just add the project name here, as you likely realized per https://github.com/darshan-hpc/darshan-logs/issues/15

Reply:
Good idea. You could probably make `cibuildwheel` just clone the repo and install it from source, but we should just make a PyPI package probably. I'll make an issue to track that.